packages/cli/package.json

```diff
@@ -45,7 +45,8 @@
     "prepack": "node -r tsconfig-paths/register ../../node_modules/.bin/oclif-dev manifest",
     "version": "node -r tsconfig-paths/register ../../node_modules/.bin/oclif-dev readme && git add README.md",
     "cli": "node -r tsconfig-paths/register bin/run",
-    "cli:debug": "node --inspect-brk -r tsconfig-paths/register bin/run"
+    "cli:debug": "node --inspect-brk -r tsconfig-paths/register bin/run",
+    "cli:example": "npm run cli -- mock -d ../../examples/petstore.oas3.json"
```

Comment:
Can we remove this command?

Reply:
Sure. I suggest after we've integrated some tests.

Reply:
https://github.com/stoplightio/prism/pull/384/commits/7e238f9a9ebf9ad6c28594e6ce1abe28108e45ee